test/ex_doc/formatter/html/erlang_test.exs

```diff
@@ -36,10 +36,10 @@ defmodule ExDoc.Formatter.HTML.ErlangTest do
              ~s|-spec</span> foo(<a href=\"#t:t/0\">t</a>()) -&gt; <a href=\"#t:t/0\">t</a>().|
 
     assert html =~
-             ~s|-type</span> t() :: <a href=\"https://www.erlang.org/doc/man/erlang.html#type-atom\">atom</a>().|
+             ~s|-type</span> t() :: <a href=\"https://www.erlang.org/doc/man/erlang.html#t:atom/0\">atom</a>().|
 
     assert html =~
-             ~s|-type</span> t2() :: #rec{k1 :: <a href=\"https://www.erlang.org/doc/man/uri_string.html#type-uri_string\">uri_string:uri_string</a>(), k2 :: <a href=\"https://www.erlang.org/doc/man/uri_string.html#type-uri_string\">uri_string:uri_string</a>() \| undefined}.|
+             ~s|-type</span> t2() :: #rec{k1 :: <a href=\"https://www.erlang.org/doc/man/uri_string.html#t:uri_string/0\">uri_string:uri_string</a>(), k2 :: <a href=\"https://www.erlang.org/doc/man/uri_string.html#t:uri_string/0\">uri_string:uri_string</a>() \| undefined}.|
```

Comment:
✅ https://www.erlang.org/doc/man/uri_string.html#t:uri_string/0 redirects to https://www.erlang.org/doc/apps/stdlib/uri_string.html#t:uri_string/0

Reply:
Should we update these links as well or is it OK to rely on www.erlang.org to always perform this redirect?

Reply:
I would say let's go ahead and link to the new page! Especially as I assume it will be less code? But we can do it in a next PR. :)